google/api_core/client_info.py

```diff
@@ -31,8 +29,10 @@
 _GRPC_VERSION: Union[str, None]
 
 try:
-    _GRPC_VERSION = pkg_resources.get_distribution("grpcio").version
-except pkg_resources.DistributionNotFound:  # pragma: NO COVER
+    import grpc
+
+    _GRPC_VERSION = grpc.__version__
```

Comment:
https://github.com/googleapis/python-api-core/blob/caba5f22b35305531626bafc4e0041619208ca26/setup.py#L38

Confirmed. The minimum version for `grpcio` in setup.py is `1.33.2` which was [published in 2020](https://github.com/grpc/grpc/releases/tag/v1.33.2). 

grpc supported the `grpc.__version__` feature since [2018](https://github.com/grpc/grpc/pull/17580)